hivemq-edge/src/test/java/com/hivemq/adapter/AdapterConfigModelTest.java

```diff
@@ -102,4 +104,92 @@ static class AdapterConfiguration implements ProtocolAdapterConfig {
             return "id";
         }
     }
+
+    private static class TestToMqttMapping implements PollingContext {
```

Comment:
I think you can use the SimulationToMqttMapping, instead of creating a new class.

Reply:
wasnt me